### PR TITLE
Improve graphql usage

### PR DESCRIPTION
**Pull request by WickyNilliams**
**DO NOT MERGE UNTIL API & SCHEMA MERGED/DEPLOYED**

### What is the context of this PR?
* improves tests for queries and mutations
* use input types
  * update mutations to have single argument and use input types
* use readFragment/writeFragment everywhere we can
* use `newId` field
  * alias to `id`, to limit scope of required changes
  * updated any code that treated id as integer

### How to review 
review code, test against associated PRs.

### CODE REVIEW

**Comment by WickyNilliams** [comment]
[Improve GraphQL usage (5)](https://trello.com/c/8bkFSzmj/187-improve-graphql-usage-5)

File: .storybook/webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+const path = require("path");
```
Comment (samiwel) [inline .storybook/webpack.config.js]: Does `path` need to be here?

File: src/graphql/answerFragment.graphql [hunk 2]
```diff
@@ -1,12 +1,7 @@
-fragment Answer on Answer {
-  id
+fragment BasicAnswer on Answer {
```
Comment (samiwel) [inline src/graphql/answerFragment.graphql]: Why have we named this `BasicAnswer` when we're getting the option Ids on `MultipleChoiceAnswer`?

Comment (WickyNilliams) [reply src/graphql/answerFragment.graphql]: Hmmm good point, didn't think of it being an overloaded term. I needed a unique name that wouldn't clash with https://github.com/ONSdigital/eq-author/blob/20bbcf0ab428e3790ca74c84e465d675f0998a7a/src/graphql/fragments/answer.graphql. 

Any ideas on a naming convention? I was trying to signify you're requesting only IDs, and not the whole thing

File: .storybook/webpack.config.js (hunk 1 above)
Comment (WickyNilliams) [reply .storybook/webpack.config.js]: Nope! Forgot to remove it :)

File: src/graphql/answerFragment.graphql (hunk 2 above)
Comment (samiwel) [reply src/graphql/answerFragment.graphql]: `AnswerFragment`?

**Pull request merged**